apps/i18n/common/en_us.json

```diff
@@ -9,6 +9,10 @@
   "addStudentsSyncThirdParty": "Sync my list of students from an existing classroom section in a third party tool",
   "addStudentsToSectionInstructions": "Choose how you want to add your students:",
   "administrator": "Administrator",
+  "administratorResourcesButton": "View resources",
+  "administratorResourcesDescription": "View resources on how to get started or expand computer science opportunities.",
+  "administratorResourcesHeading": "Administrators: Expand computer science in your school or district",
```

Comment:
Some of these are duplicate strings. `yourSchoolAdminButton` and `yourSchoolAdminDesc`. Maybe consolidate.